go.sum

```diff
@@ -15,6 +15,8 @@ github.com/chef/go-chef v0.0.0-20191115205148-7ac24fba7a27 h1:XGQlXv6Gwd2+tBBXrp
 github.com/chef/go-chef v0.0.0-20191115205148-7ac24fba7a27/go.mod h1:ft8oU7s0ktse9xzZO4n8wxTz6EKx+V+fuhYxktqXvG8=
 github.com/chef/go-libs v0.0.0-20191113223254-d2a05fa5636d h1:0w673WjR7dQUn1wcNKhMKfQ06GQQvik6uYdcwaWINsg=
 github.com/chef/go-libs v0.0.0-20191113223254-d2a05fa5636d/go.mod h1:aGHwP5PXYdD0hVq+jodILnSqB/b04EhS/XCoqhE2PrA=
+github.com/chef/go-libs v0.0.0-20191216182352-aa2fc405b05f h1:kBujwkOBM7smuh1xCKHrUTdkZbNC0IH30zzDmTNYp8U=
```

Comment:
I hadn't noticed this before - does it keep all known versions of the `go-libs` library?

Reply:
yeah, It is kinda odd and confusing, we can clean it up by running `go mod tidy`, I will try to do that.